### PR TITLE
Added Appointments screen and care models

### PR DESCRIPTION
**Pull request by CleshanW**
# Pull Request-Trello card number:  34
Changes:
* UI Added Appointments Tab
* Added Appointments and care model expandable components

Notes:
* Checkbox doesn't completely work (that's fine for now)
* 
## CheckList
* [ ] Code must be commented for any non-trivial code
* [ ] UI must be aesthetically pleasing on iOS & Android
* [ ] Functions require full documentation (READMEs, in-line comments, docstrings and type hinting) for any part that is not already outlined by the code itself.
* [ ] Meets non-functional requirements
* [ ] Pipeline has passed and any additional unit tests have been added
* [ ] Code Reviewed by
* [ ] Tested by

### CODE REVIEW

File: src/screens/AppointmentsScreen.js [hunk 1]
```diff
@@ -1,10 +1,327 @@
...
+        />
+      );
+    }
+
+    if (props.val == 'Phone') {
+      return <Text style={styles.Text}>phone dog</Text>;
+    }
+  };
+
+  // function ContactCardPhone({ number }) {
```
Comment (fi-crowe) [inline src/screens/AppointmentsScreen.js:181]: commented out code?

File: src/screens/AppointmentsScreen.js [hunk 2]
```diff
@@ -1,10 +1,327 @@
...
+  // };
+
+  const AppointmentComponents = ({ props }) => {
+    if (props.val == 'Description') {
+      var description = props.descriptionVal;
+      return <Text style={styles.Text}>{description}</Text>;
+    }
+    if (props.val == 'CheckBox') {
+      return (
+        <Checkbox
```
Comment (fi-crowe) [inline src/screens/AppointmentsScreen.js:167]: I think we are using the favourite button now like for practitioners' and facilities page to make most consistent & visible :)

**Comment by vishaljak** [comment]
Not sure if it's only me, but I can't run this. Gives me an error.

Also, please try and include comments for some parts of code which may be hard to understand. Thanks!!

File: src/screens/AppointmentsScreen.js (hunk 2 above)
Comment (vishaljak) [reply src/screens/AppointmentsScreen.js:167]: Not sure if it's only me, but I can't run this. Gives me an error.

Also, please try and include comments for some parts of code which may be hard to understand. Thanks!! 😄

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     "@firebase/auth": "^0.20.2",
     "@firebase/firestore": "^3.4.9",
     "@react-native-async-storage/async-storage": "~1.15.0",
+    "@react-native-community/checkbox": "^0.5.12",
```
Comment (fi-crowe) [inline package.json:20]: looks like we have multiple dependencies for checkboxes - do we need them at all now we will be using the FavouriteButton component?

**Review by fi-crowe — APPROVED** [review]

**Review by vishaljak — APPROVED** [review]

**Pull request merged**